ucm/sof-glkda7219max/sof-glkda7219max.conf

```diff
@@ -0,0 +1,32 @@
+SectionUseCase."HiFi" {
+	File "HiFi.conf"
+	Comment "Default"
+}
+
+SectionUseCase."Hdmi1" {
+	File "Hdmi1.conf"
+	Comment "Play to Hdmi1"
+}
+
+SectionUseCase."Hdmi2" {
+	File "Hdmi2.conf"
+	Comment "Play to Hdmi2"
+}
+
+SectionUseCase."Hdmi3" {
+	File "Hdmi3.conf"
+	Comment "Play to Hdmi3"
+}
+
+SectionDefaults [
+	cdev "hw:sofglkda7219max"
+	cset "name='Headphone Switch' off"
+	cset "name='Headphone Jack Switch' off"
+	cset "name='Mic Volume' 50"
+	cset "name='Out DACL Mux' DAIL"
+	cset "name='Out DACR Mux' DAIR"
+	cset "name='Out DAIL Mux' ADC"
+	cset "name='Out DAIR Mux' ADC"
+	cset "name='Gain Ramp Rate' 1"
+	cset "name='Playback Digital Volume' 115"
+]
```

Comment:
this is usually at the start?

Reply:
@plbossart  we are starting to use separated profile, and many ucms in sof_ucm1 repo are adapted to separated profile.
In the separated profile, we have one main profile to include all other profiles, and default values is defined in the main profile.
I think the sof_ucm1 branch is used to store our old ucm1 profiles, ucm2 is in other branch. and Libin already has a pending UCM2 PR.